Dockerfile.openhabian-ARMv6hf-emulated

```diff
@@ -1,38 +1,29 @@
-FROM minimum2scp/systemd-buster
-#FROM balenalib/intel-nuc-debian-node:latest-buster
-#FROM balenalib/raspberry-pi2-debian-node:latest-buster
-
-# Docker concept based on
-# https://www.balena.io/docs/reference/base-images/base-images/#installing-your-own-initsystem
+# RPi1, RPi0W (ARMv6hf arch)
+FROM balenalib/raspberry-pi-debian-node:latest-buster
+ 
+RUN [ "cross-build-start" ]
 
 ENV container docker
 ENV INITSYSTEM on
 ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
 
-# install base packages usually installed by Raspbian image and first-boot.bash
-# delete temporary files to minimize build size
 RUN apt-get update -qq \
-    && apt-get install -y -qq --no-install-recommends \
-    systemd systemd-sysv \
-    git wget apt-utils jq virt-what \
-    python python-pip python3 python3-pip \
+    && apt-get install -y -qq --no-install-recommends systemd systemd-sysv git wget python python-pip python3 python3-pip apt-utils jq virt-what \
     && rm -rf /var/lib/apt/lists/*
-
-# install BATS framework for testing
```

Comment:
why removing comments?

Reply:
Huh, Dockerfile.openhabian? Doesn't exist any more, was rm'ed/duplicated after #683 ...

Reply:
But it was the template for the others, so git detected it as renaming instead of new file for one of them... 
Anyway, comment is gone from all dockerfiles...